src/main/java/org/cy3sbml/SBMLReaderTask.java

```diff
@@ -197,7 +197,6 @@ public CyNetworkView buildCyNetworkView(final CyNetwork network) {
         }
 
         // layout
-        if (doLayout) {
```

Comment:
This should not be changed! This introduces a bug with the tunables. This is not relevant for the pull request, please update the changes to develop

Reply:
Please do not change this file in the pull request! I.e. revert the changes on this file. The 
```
if (doLayout) {

}
```
is required.